source/main/providers/config-provider.js

```diff
@@ -185,6 +185,7 @@ class ConfigProvider extends EventEmitter {
         'enableTableHelper': true, // Enable the table helper plugin
         'indentUnit': 4, // The number of spaces to be added
         'countChars': false, // Set to true to enable counting characters instead of words
+        'editorMode': 'default', // Can be default, vim, emacs
```

Comment:
Maybe rename that to `insertMode`? Because as it is currently, in the codebase it would look like `global.config.get('editor.editorMode')` which sounds repetitively; and at least my gut feeling tells me that `global.config.get('editor.inputMode')` (or insert, something like that) tells more about what the config var is for?